uint32 order
uint32 numDOF
string type
ramp_planner_new/Coefficient[] coefficients
float64 resolution
bool active